Makefile

```diff
@@ -0,0 +1,15 @@
+SHELL := /bin/bash
+
+init:
+	dep ensure -v
+	go install ./vendor/...
+
+up_docker_server:
+	test -n "$$(docker ps -a --format {{.Names}} | grep -o dbr-clickhouse-server)" && docker rm -f dbr-clickhouse-server; \
+	docker run -p 127.0.0.1:8123:8123 --name dbr-clickhouse-server -d yandex/clickhouse-server;
```

Comment:
IMHO it would be better to add option --rm=true  to remove container after stop

Reply:
wow! i didn't know about that flag